photutils/tests/test_psfs.py

```diff
@@ -19,20 +19,108 @@
 @pytest.mark.parametrize('width', widths)
 def test_subpixel_gauss_psf(width):
     """
-    Test subpixel accuracy of Gaussian PSF by checking the peak
-    amplitude.
+    Test subpixel accuracy of Gaussian PSF by checking the sum of pixels.
     """
-    gauss_psf = GaussianPSF(width)
+    gauss_psf = IntegratedGaussianPRF(width)
     y, x = np.mgrid[-10:11, -10:11]
-    assert_allclose(gauss_psf(x, y).max(), 1)
+    assert_allclose(gauss_psf(x, y).sum(), 1)
 
 
 @pytest.mark.skipif('not HAS_SCIPY')
 @pytest.mark.parametrize('sigma', sigmas)
 def test_gaussian_psf_integral(sigma):
     """
-    Test if Gaussian PSF peak matches amplitude.
+    Test if Gaussian PSF integrates to unity on larger scales.
     """
-    psf = GaussianPSF(sigma=sigma)
+    psf = IntegratedGaussianPRF(sigma=sigma)
     y, x = np.mgrid[-100:101, -100:101]
-    assert_allclose(psf(y, x).max(), 1)
+    assert_allclose(psf(y, x).sum(), 1)
+
+@pytest.fixture(scope="module")
+def moffimg():
+    """
+    This fixture requires scipy so don't call it from non-scipy tests
+    """
+    from scipy import integrate
+    from astropy.modeling.models import Moffat2D
+
+    mof = Moffat2D(alpha=4.8)
+
+    # this is the analytic value needed to get a total flux of 1
+    mof.amplitude = (mof.alpha-1)/(np.pi*mof.gamma**2)
+
+    # first make sure it really is normalized
+    assert (1 - integrate.dblquad(mof, -10, 10, lambda x: -10, lambda x: 10)[0]) < 1e-6
+
+    # now create an "image" of the PSF
+    xg, yg = np.meshgrid(*([np.linspace(-2, 2, 100)]*2))
+    img = mof.render(coords=(xg, yg))
```

Comment:
I think we want to try to continue supporting the current LTS. Even if the PSF stuff is rather experiemntal, there are plenty of "real users" on the aperture and star-finder side of things.  Forcing them to upgrade to 1.1 might be more of an issue.

But this particular use of `render` is really trivial and I now realize unnecessary.  I just checked now and it turns out ``mof(xg,yg)`` works so we can just change to that.